setup.py

```diff
@@ -26,6 +28,13 @@
 author = re.search("__author__ = '([^']+)'", rwsinit).group(1)
 version = re.search("__version__ = '([^']+)'", rwsinit).group(1)
 
+INSTALL_REQUIRES = ['requests', 'lxml', 'httpretty', 'six', 'click', 'faker']
```

Comment:
Could we use `enum34; python_version < '3.4'` ?

Reply:
great suggestion, I didn't know it was possible